doc/source/satimg_class.md

```diff
@@ -0,0 +1,54 @@
+---
+file_format: mystnb
+kernelspec:
+  name: geoutils
+---
+(satimg-class)=
+
+# The geo-image ({class}`~geoutils.SatelliteImage`)
+
+Below, a summary of the {class}`~geoutils.SatelliteImage` object and its methods.
+
+## Object definition
+
+A {class}`~geoutils.SatelliteImage` is a subclass of {class}`~geoutils.Raster` that contains all its main attributes, and retains additional ones:
+- a {class}`numpy.datetime64` as {class}`~geoutils.SatelliteImage.datetime`, and
+- several {class}`strings<str>` for`~geoutils.SatelliteImage.satellite`,  {class}`~geoutils.SatelliteImage.sensor`, {class}`~geoutils.SatelliteImage. version`, {class}`~geoutils.SatelliteImage.tile_name` and {class}`~geoutils.SatelliteImage.product`.
```

Comment:
`~geoutils.SatelliteImage.satellite` does not display well

Reply:
Fixed